### PR TITLE
Je/custom run with all backends

### PR DESCRIPTION
**Pull request by emord**
Just look at first/third commit. second commit was changing windows line endings to unix to match rest of hq and pep8 was yelling at me.

Noticed this when looking at test output from https://github.com/dimagi/commcare-hq/pull/14865 Since this is custom code and we know that they will only be on sql domains, we shouldn't test on couch.

before: `Ran 124 tests in 522.590s`

after: `Ran 124 tests in 214.298s`

@czue @millerdev

### CODE REVIEW

**Review by benrudolph — APPROVED** [review]
cha-ching. that reasoning makes sense to me

**Comment by benrudolph** [comment]
maybe this'll put us over the edge for 3 runners

File: custom/ucla/tests.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 from corehq.apps.users.models import WebUser
-from corehq.form_processor.tests.utils import run_with_all_backends
 from corehq.util.test_utils import create_test_case
 from custom.ucla.api import ucla_message_bank_content
 
 Reminder = namedtuple('Reminder', ['domain', 'schedule_iteration_num', 'current_event_sequence_num'])
 Handler = namedtuple('Handler', ['events'])
 
 
+@override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
```
Comment (czue) [inline custom/ucla/tests.py]: confirming you checked this domain?

**Review by czue — APPROVED** [review]
cool, looks good. just asked a dumb question since it was the only one I wasn't sure myself was on SQL.

File: custom/ucla/tests.py (hunk 1 above)
Comment (emord) [inline custom/ucla/tests.py]: Yep. it's for a new domain. (I wrote this code in the past month)

**Pull request merged**